src/styles/nav.scss

```
@@ -0,0 +1,79 @@
+@import "../utils/variables.scss";
+
+* {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+:root {
+  --color: $buttonAndActivesLinksColor;
+}
+
+.container {
```

since header is a BEM block - move it to a separate file
```suggestion
.header {
```